packages/scripts/src/remove-cases.js

```diff
@@ -384,7 +389,20 @@ const getReferencesPrefixedWith = async (startsWith) => {
  * @returns {Promise<void>}
  */
 export const removeAllTrainingCases = async () => {
-	// const references = await getReferencesPrefixedWith('TRAIN0110004');
 	const references = await getReferencesPrefixedWith('TRAIN');
 	await removeCases(references);
 };
+
+/**
+ * Remove all specified cases and associated records
+ *
+ * @returns {Promise<void>}
+ */
+export const removeSpecifiedCases = async () => {
+	if (specifiedCases.length > 0) {
+		console.log('Specified cases to be removed:');
+		specifiedCases.forEach((reference) => console.log('- ' + reference));
+		console.log('');
+	}
+	await removeCases(specifiedCases);
```

Comment:
Does `removeCases` do anything if the list `specifiedCases` list is empty? If not, might be best to just return early from this function when the list is empty, like:

```suggestion
	if (specifiedCases.length === 0) {
		return;
	}
	
	console.log('Specified cases to be removed:');
	specifiedCases.forEach((reference) => console.log('- ' + reference));
	console.log('');

	await removeCases(specifiedCases);
```

Reply:
remove cases will report that there are 0 cases to remove and I thought that would be more useful.